src/diagram/Diagram.ts

```diff
@@ -0,0 +1,151 @@
+import { Config } from "dompurify";
+import { Field } from "./Field";
+import { Configuration } from "../config/Configuration";
+import { BooleanOption, EnumOption, RangeOption } from "../config/Option";
+
+interface FieldPair {
```

Comment:
Change the interface name to `MementoFieldPair`

Reply:
updated.